### DISCUSSION TITLE
Linux packages only

### DISCUSSION THREAD

**Discussion by ZehavaG** [python-poetry/poetry#9541] [Q&A]
Is there a way for me to tell poetry to only look for linux packages when doing "poetry add" or "poetry install"?

**Comment by Secrus** [comment]
No, that's defeats the purpose of having cross-platform lock file. If you want to mark a depenendency to only be installed on linux systems, you can add condition i dependency definition: [docs](https://python-poetry.org/docs/dependency-specification/#expanded-dependency-specification-syntax), [example](https://github.com/python-poetry/poetry/blob/a7d234cc6afaee1ec6d389cf4499ebf36e8fa467/pyproject.toml#L58)
